### DISCUSSION TITLE
Redmine Multicore

### DISCUSSION THREAD

**Discussion by Nyralei** [docker-library/redmine#323] [General]
Hello. I'm using this Redmine image in Kubernetes cluster.
For some reason ruby process is using not more than 100% of one CPU core. Is there some sort of flag that needs to be set for it to use more cores?
Requests are set to CPU: 4.

**Comment by tianon** [comment]
This image uses `rails server` by default, which as far as I know unfortunately does not implement any concurrency.

**Reply by Nyralei** [reply]
Moved to bitnami/redmine image and it spawns multiple ruby processes.
Command to start server is `/opt/bitnami/redmine/vendor/bundle/ruby/3.1.0/bin/passenger start --user redmine -e production -p 3000`
